### PR TITLE
normalize schema to GHE

### PR DESCRIPTION
**Pull request by lwsanty**
close [[DEV-296]]

Signed-off-by: lwsanty <lwsanty@gmail.com>

[DEV-296]: https://athenianco.atlassian.net/browse/DEV-296

### CODE REVIEW

File: github/ghnodes/codegen/codegen.go [hunk 1]
```diff
@@ -127,6 +129,12 @@ var (
 		"UnassignedEvent": {
 			"assignable": {},
 		},
+		"Team": {
+			"databaseId": {},
```
Comment (dennwc) [inline github/ghnodes/codegen/codegen.go:133]: Please comment why.

File: github/ghnodes/types_gen.go [hunk 2]
```diff
@@ -115,7 +115,6 @@ var (
 			{Name: "descriptionHTML", Type: "String", ID: "description_html"},
 			{Name: "email", Type: "String", ID: "email"},
 			{Name: "id", Type: "ID", ID: "id"},
```
Comment (dennwc) [inline github/ghnodes/types_gen.go:117]: Shouldn't `databaseId` disappear from `Team` as well?

File: github/ghnodes/codegen/codegen.go (hunk 1 above)
Comment (lwsanty) [reply github/ghnodes/codegen/codegen.go:134]: Done.

File: github/ghnodes/types_gen.go (hunk 2 above)
Comment (lwsanty) [reply github/ghnodes/types_gen.go:117]: Discussed in DM.

**Pull request merged**